.ci/validation.sh

```diff
@@ -216,12 +216,17 @@ no-error-pmd)
   echo "CS_version: ${CS_POM_VERSION}"
   mvn -e --no-transfer-progress clean install -Pno-validations
   echo "Checkout target sources ..."
-  checkout_from "https://github.com/pmd/build-tools.git"
+  #  checkout_from "https://github.com/pmd/build-tools.git"
+  checkout_from "https://github.com/kkoutsilis/build-tools.git"
```

Comment:
remove all unrelated changes from your PR, too much files are changed.

Reply:
i only see this right now im wondering why. Thanks

Reply:
you did something weird on your local , and you pulled somebody else changes to your commit.
just redo commit.

Reply:
lets complete as least this checker then, if not unused method check is possible. Check return and throw is possible. @romani